### PR TITLE
Remove duplicate mongo services in docker-compose.yml

### PR DESCRIPTION
**Pull request by sgundlapalli11**
I think this is what was wanted from this issue. Also not entirely sure if a container name is required or if the ordering of the containers matter. 

#61

### CODE REVIEW

**Comment by jasonify** [comment]
Thanks for looking into this :)

File: docker-compose.yml [hunk 1]
```diff
@@ -1,12 +1,6 @@
 version: '2'
 
 services:
-  mongo:
-    image: mongo:3.5
-    container_name: sedaily-mongo
```
Comment (jasonify) [inline docker-compose.yml]: Do you think we could keep the container name?

Comment (sgundlapalli11) [reply docker-compose.yml]: Yep, just added it back.

Comment (jasonify) [reply docker-compose.yml]: Awesome !

**Pull request merged**